apps/coa/fields/index.js

```diff
@@ -199,5 +199,54 @@ module.exports = {
         value: 'legal-details'
       }
     ]
+  },
+
+  'old-address': {
+    mixin: 'radio-group',
+    validate: 'required',
+    isPageHeading: true,
+    options: [
+      {
+        value: 'Yes',
+        toggle: 'old-postcode',
+        child: 'input-text'
+      },
+      {
+        value: 'No'
+      }
+    ]
+  },
+
+  'old-postcode': {
+    mixin: 'input-text',
```

Comment:
`mixin: 'input-text'` can be removed, default field when declared is input-text

Reply:
changes made to remove the mixin for that field